operator_toolkit/stop_operation.py

```diff
@@ -0,0 +1,79 @@
+import argparse
+from http import HTTPStatus
+
+import requests
+from requests.auth import HTTPBasicAuth
+
+from common import (
+    CLI_MESSAGE,
+    sites,
+    AXIOM_API_AUTH_USER,
+    AXIOM_API_AUTH_PASSWORD
+)
+
+
+def stop_operation(site, operation_type, stop: str):
+    """Recieves a site id and stops the mode operation of the whole operation
+    on that site, taking in consideration the operation type.
+
+    :param str site: site id to process.
+    :param str operation_type: type of service operation to process.
+    :param str stop: kind of stop operation to process
+    :return: returns the result of the operation.
+    :rtype: str
+    """
+    stop_url = "modes/stop" if stop == "mode" else "stop"
+    url = f'http://{operation_type}-{site}.axiomcloud.io/{stop_url}'
```

Comment:
```suggestion
    url = f'https://{operation_type}-{site}.axiomcloud.io/{stop_url}'
```
I think the request should use `https`